### ISSUE TITLE
Allow some way to add additional hosts

### ISSUE THREAD

**Issue by jabl** [CSC-IT-Center-for-Science/ansible-role-pxe_config#23]
Currently there is a hardcoded list of groups to use when generating entries in /etc/hosts, namely hosts_file_{pxe,grid,login,admin,install}_group_to_populate. It would be useful to have yet another group, say, for adding external hosts with interfaces in the cluster internal net (e.g. file server appliances) so that one could refer to these by DNS names rather than IP's.

**Comment by martbhell** [comment]
Agreed. I think this is a lost feature in the migration from FGI to ansible/FGCI > bug.

**Comment by martbhell** [comment]
Created a PR for this. What do you think? Basically just added another in the template.

https://travis-ci.org/CSC-IT-Center-for-Science/ansible-role-pxe_config/jobs/155010382#L3773 is how the hosts file looks after configuring it.

**Comment by martbhell** [comment]
Copy pasted without looking too much. Waiting for another test build.

**Comment by martbhell** [comment]
Now [1](https://travis-ci.org/CSC-IT-Center-for-Science/ansible-role-pxe_config/jobs/155014845#L3807) looks a it better I think

<pre>
10.44.55.1 ibsw1 ibsw1.int.fgci.csc.fi
10.55.66.1 ibsw1-ib ibsw1-ib.int.fgci.csc.fi
10.44.55.20 ethsw20 ethsw20.int.fgci.csc.fi
</pre>

With this group in the inventory:

<pre>
[switches]
ibsw1 int_ip_addr=10.44.55.1 ib_ip_addr=10.55.66.1
ethsw20 int_ip_addr=10.44.55.20
</pre>

and this variable:

<pre>
     - hosts_file_extra_group_to_populate: "{{ groups.switches }}"
</pre>

**Comment by jabl** [comment]
Looks great, exactly what I had in mind!

Once this is merged, you might want to add something to fgci-ansible/examples too.

**Comment by martbhell** [comment]
Merged into devel in https://github.com/CSC-IT-Center-for-Science/fgci-ansible/commit/e37d8d7a559a0277b9213424def7aedc9595b56b
